package.json

```diff
@@ -45,15 +45,13 @@
     "ember-source": "~2.15.0",
     "ember-truth-helpers": "^1.2.0",
     "ember-welcome-page": "^3.0.0",
-    "loader.js": "^4.2.3",
-    "node-sass": "^3.4.2"
+    "loader.js": "^4.2.3"
   },
   "engines": {
     "node": "^4.5 || 6.* || >= 7.*"
   },
   "dependencies": {
     "ember-cli-dropzonejs": "^1.0.1",
-    "ember-cli-materialize": "^0.22.0-beta.0",
-    "ember-font-awesome": "^4.0.0-alpha.10"
```

Comment:
Why have you removed the font-awesome dependency? That's probably why it's not working anymore